### PR TITLE
Added UI views

### PR DESCRIPTION
**Pull request by isNan909**
Added UI views with some auth file structures.

### CODE REVIEW

File: src/pages/Login.svelte [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+  import Button from "../components/Button.svelte";
+  import Input from "../components/Input.svelte";
+  import { SIGN_UP_PAGE } from "../constants/routes";
+
+  let email;
+  let password;
+
+  async function login() {
+    console.log(email, password);
```
Comment (su-sh) [inline src/pages/Login.svelte]: Shall we remove this console log?

File: src/pages/Signup.svelte [hunk 2]
```diff
@@ -0,0 +1,116 @@
+<script>
+  import axios from "axios";
+  import { user } from "../stores/store";
+  import { push, link } from "svelte-spa-router";
+
+  import Button from "../components/Button.svelte";
+  import Input from "../components/Input.svelte";
+  import { LOGIN_PAGE } from "../constants/routes";
```
Comment (su-sh) [inline src/pages/Signup.svelte]: Let's group and order the imports.

File: src/pages/Signup.svelte [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  let password;
+
+  async function signup() {
+    try {
+      await axios.post("/api/auth/signup", { email, username, password });
+      $user = data.user;
+      push("/home");
+    } catch (e) {
+      if (e) {
+        console.log(e);
```
Comment (su-sh) [inline src/pages/Signup.svelte]: .

File: src/components/Button.svelte [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+          <path
+            fill-rule="evenodd"
+            d="M5 9V7a5 5 0 0110 0v2a2 2 0 012 2v5a2 2 0 01-2 2H5a2 2 0 01-2-2v-5a2 2 0 012-2zm8-2v2H7V7a3 3 0 016 0z"
+            clip-rule="evenodd" />
+        </svg>
+      </span>
+      <slot>Submit</slot>
+    </button>
+  </div>
+
```
Comment (su-sh) [inline src/components/Button.svelte]: Let's add a new line at the end of the file.

**Review by su-sh — COMMENTED** [review]
@ishan-me, @Revortni & @sumitdhungana14 Thanks for the contribution.

I have added a few minor comments.

Can you please have a look at it?

File: src/pages/Signup.svelte (hunk 3 above)
Comment (isNan909) [reply src/pages/Signup.svelte]: I think we don't need if inside a catch block.

**Review by su-sh — APPROVED** [review]

**Comment by su-sh** [comment]
LGTM 👍 

Thanks for the contribution @ishan-me @sumitdhungana14 @ShishirBhandari @Revortni.

**Pull request merged**